### PR TITLE
Add name and description tests.

### PR DESCRIPTION
**Pull request by BigBlueHat**
- Move name and description tests into new section.
- Move name/description fixtures into folder.
- Test name/description on main credential object.

### CODE REVIEW

File: tests/10-vcdm2.js [hunk 1]
```diff
@@ -299,8 +299,95 @@ describe('Types', function() {
...
+// to cover `name` and/or `description` anywhere they appear.
+describe('Names and Descriptions', function() {
+  setupMatrix.call(this);
+  for(const [name, implementation] of match) {
+    const endpoints = new TestEndpoints({implementation, tag});
+
+    describe(name, function() {
+      beforeEach(addPerTestMetadata);
+
+      const fixturePath = './input/names-and-descriptions';
```
Comment (aljones15) [inline tests/10-vcdm2.js:314]: this is fine for now, but if this suite endpoints with a config dir making `fixturePath` configurable in the future is a good idea.

File: tests/10-vcdm2.js [hunk 2]
```diff
@@ -299,8 +299,95 @@ describe('Types', function() {
...
+          // eslint-disable-next-line max-len
+          `${fixturePath}/credential-description-language-direction-en-ok.json`));
+        await endpoints.issue(require(
+          `${fixturePath}/credential-multi-language-description-ok.json`));
+        await assert.rejects(endpoints.issue(require(
+          `${fixturePath}/credential-description-extra-prop-en-fail.json`)));
+      });
+
+      // On `issuer` as in the example at https://w3c.github.io/vc-data-model/#example-usage-of-the-name-and-description-property-0
+      // @link https://w3c.github.io/vc-data-model/#names-and-descriptions:~:text=If%20present%2C%20the%20value%20of%20the%20name%20property%20MUST%20be%20a%20string%20or%20a%20language%20value%20object%20as%20described%20in%2011.1%20Language%20and%20Base%20Direction.
```
Comment (aljones15) [inline tests/10-vcdm2.js:353]: when will the links be moved into the suite itself and displayed in the report or does your reporter function look at jsdoc comments?

**Review by aljones15 — APPROVED** [review]
LGTM

File: tests/10-vcdm2.js (hunk 2 above)
Comment (BigBlueHat) [reply tests/10-vcdm2.js:353]: It's in the next PR I plan to submit.

Comment (BigBlueHat) [reply tests/10-vcdm2.js:353]: https://github.com/w3c/vc-data-model-2.0-test-suite/pull/77

**Pull request merged**